commands/commands.go

```diff
@@ -57,6 +57,17 @@ func Exit(format string, args ...interface{}) {
 	os.Exit(2)
 }
 
+func ExitWithError(err error) {
+	if Debugging || lfs.IsFatalError(err) {
+		Panic(err, err.Error())
+	} else {
+		if inner := lfs.GetInnerError(err); inner != nil {
```

Comment:
Technically err.InnerError can be another `lfs.wrappedError` so maybe this should be a loop until that's nil?